packages/expo-sharing/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 ### 🐛 Bug fixes
 
+### ⚠️ Notices
+
+- Changed the return type of the `shareAsync` function. The promise now resolves to `void` instead of an empty object. ([#18019](https://github.com/expo/expo/pull/18019) by [@barthap](https://github.com/barthap))
+
```

Comment:
It's a breaking change IMO 🤔

Reply:
It's unfortunate that such a small change would be marked as breaking 😞 I doubt that anybody ever used the return value.

Reply:
Scope of a change is not a problem, but is it a breaking change indeed (API changed right?), so we need to be consistent in categorizing PRs under proper section.